resources/html/terms.html

```diff
@@ -419,19 +402,19 @@
             }
         </style>
     </head>
-    <body class="c9 c13">
-        <p class="c14 title" id="h.1h19t7261en8">
+    <body class="c5">
+        <p class="c26 title" id="h.1h19t7261en8">
             <span class="c23"
                 >Terms and conditions for Concordium Desktop Wallet</span
             >
         </p>
-        <h1 class="c20">
-            <span class="c6">Effective from June 09, 2021</span>
+        <h1 class="c11">
+            <span class="c3">Effective from June 09, 2021</span>
```

Comment:
This date has to be updated as the document has changed.

Reply:
Let's do this one parallel to the docs change...

Reply:
I've notified Lotte, and she's OK with doing the change in parallel.